shanoir-ng-datasets/src/main/java/org/shanoir/ng/dataset/service/DatasetServiceImpl.java

```diff
@@ -367,6 +367,7 @@ public List<Dataset> findByExaminationId(Long examinationId) {
 
 	@Override
 	public List<Object[]> queryStatistics(String studyNameInRegExp, String studyNameOutRegExp, String subjectNameInRegExp, String subjectNameOutRegExp) throws Exception {
+		LOG.error("query statistics : " + studyNameInRegExp);
```

Comment:
not LOG.error please and please log all RegExp to have the full info of the query in the logs